src/assets/wise5/services/peerGroupService.ts

```diff
@@ -69,4 +69,14 @@ export class PeerGroupService {
   removeWorkgroupFromGroup(workgroupId: number, groupId: number): Observable<any> {
     return this.http.delete(`/api/peer-group/membership/${groupId}/${workgroupId}`);
   }
+
+  retrieveDynamicPromptStudentData(
+    peerGroupId: number,
+    nodeId: string,
+    componentId: string
+  ): Observable<any> {
```

Comment:
Can we define a PeerGroupData and use it instead of any? I think it would help make the code a bit clearer.

interface PeerGroupData {
  annotation: any;
  studentWork: any;
  workgroup: any;
}